modules/ROOT/pages/to-find-info.adoc

```diff
@@ -1,27 +1,10 @@
-= Find Asset Information
+= Search for Assets
 ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
-Exchange helps you find assets. You can search for assets, click All Types to filter for specific assets, or combine search
-text with an All Types filter.
-
-<<index.adoc#content-types,Content Types>> lists the different types of asset.
-
-== Asset Details
-
-When you click an asset, Exchange provides detailed information:
-
-* Left navigation area - Lists additional pages, saved search terms, application names, and instances. For a REST API,
-this area lists the HTTP functions in the REST specification. For a policy, this area lists *Terms and conditions* and *APIs using this policy*.
-* Right detail panel - Lists the asset type, when the asset was created and by which organization, when the asset was last published, a list of asset patch versions, and the tags and categories of the asset. For REST APIs, the right panel lets you test the API by creating example calls, test data (mocking), and the ability to send data to the API and view the results.
-
-In the table of asset patch versions on the detail panel, each version has a drop-down menu with a *Version detail* option that shows information about that version, including who created it, when it was published, and any dependent assets.
-
-== Exchange Search
-
-Exchange Search looks for terms in:
+Anypoint Exchange search looks for terms in:
 
 * Each asset's name
 * Tags
```

Comment:
```suggestion
* Tags and Categories
```

Reply:
I think it's better to keep tags and categories as two items in this list. The old search uses different syntax for searching for them, and the old search section has different sections in this file explaining how to search for tags and how to search for categories. The new search has separate drop-down menus for them. However, if there's a strong reason to join them on one line that I'm not seeing, please tell me.

Reply:
Oh, my bad, I didn't see categories was in another item and thought we had not included them. I agree with you that keeping them separated is better 👍

Reply:
Understood. Thank you!